### ISSUE TITLE
intercept url seems to not work on macOS app

### ISSUE THREAD

**Issue by mrdavidjcole** [quip/quip-apps#4]
Hi Quip friends,

I've got `intercept_url_patterns` working well when pasting in the web app, but it doesn't seem to work on the macOS app:

![screen recording 2017-12-01 at 01 02 pm](https://user-images.githubusercontent.com/752729/33496408-19a6c7c0-d698-11e7-9a49-50e4df116e5d.gif)

This a known issue? Any info I can provide to help figure it out?

Thanks!

**Comment by plinehan** [comment]
Yes, sorry you're running into this.  This is a known issue that we're working on.

**Comment by mrdavidjcole** [comment]
Hey, no prob @plinehan. Thanks for letting me know you're on the case. 🙌
